js/nomNom.js

```diff
@@ -0,0 +1,197 @@
+	},
+	orderResultsByRank : function() {
+		fourSquare.config.foodSpots.sort(function(a,b){
+			return b.likes-a.likes;
+		});
+		populateTheCells.init(0);
+	},
+	pingFourSquare : function() {
+		var httpRequest = 'https://api.foursquare.com/v2/venues/search?client_id=5TN04INYIF1T2C2QIO24I1YV2R41KTDN0YAYIRKRANEPUAPS&client_secret=0MO2N2SNHPNYF21LGYQSTYB3USPDXXEIJXV40UFLQDF4IHAJ&v=20130815&near=' + zipCode + '&categoryId=4d4b7105d754a06374d81259&limit=31&radius=200';
+		$.ajax({
+			xhrFields: {
+				withCredentials: false
+			},
+			crossDomain: true,
+			type: "GET",
+			url: httpRequest,
+			dataType: "json",
+			success: function(json) {
+				$(json.response.venues).each(function(i){
+					fourSquare.config.foodSpots[i] = {};
+					fourSquare.config.foodSpots[i].name = this.name;
+					fourSquare.config.foodSpots[i].address = this.location.formattedAddress[0];
+					fourSquare.getNumberOfLikes(this.id, i);
+					fourSquare.getImagePath(this.id, i);
+				});
+			},
+			error: function(json) {
+				console.log("Error pinging FourSquare");
+			}
+		}).done(function(){
+			setTimeout(function(){fourSquare.orderResultsByRank();},2000);
+		});	
+	},
+	getNumberOfLikes : function(resultsId,index) {
+		var httpRequest = 'https://api.foursquare.com/v2/venues/' + resultsId + '/likes?client_id=5TN04INYIF1T2C2QIO24I1YV2R41KTDN0YAYIRKRANEPUAPS&client_secret=0MO2N2SNHPNYF21LGYQSTYB3USPDXXEIJXV40UFLQDF4IHAJ&v=20130815';
+		$.ajax({
+			xhrFields: {
+				withCredentials: false
+			},
+			crossDomain: true,
+			type: "GET",
+			url: httpRequest,
+			dataType: "json",
+			success: function(json) {
+				fourSquare.config.foodSpots[index].likes = json.response.likes.count;
+			},
+			error: function(json) {
+				console.log("Error getting Likes");
+			}
+		});	
+	},
+	getImagePath : function(resultsId,index) {
+		var httpRequest = 'https://api.foursquare.com/v2/venues/' + resultsId +'/photos?client_id=5TN04INYIF1T2C2QIO24I1YV2R41KTDN0YAYIRKRANEPUAPS&client_secret=0MO2N2SNHPNYF21LGYQSTYB3USPDXXEIJXV40UFLQDF4IHAJ&v=20130815';
+		$.ajax({
+			xhrFields: {
+				withCredentials: false
+			},
+			crossDomain: true,
+			type: "GET",
+			url: httpRequest,
+			dataType: "json",
+			success: function(json) {
+				if (json.response.photos.count === 0)
+						fourSquare.config.foodSpots[index].img = '../img/noimage.png';
+				else
+					fourSquare.config.foodSpots[index].img = json.response.photos.items[0].prefix + '200x200' + json.response.photos.items[0].suffix;
+			},
+			error: function(json) {
+				console.log("Error getting Images");
+			}
+		});	
+	}
+};
+
+var showMore = {
+	init : function() {
+		$('#show-more').on('click',function(){
+			startCell = startCell + 10;
+			showMore.loadNewData();
+		});
+	},
+	loadNewData : function() {
+		populateTheCells.init(startCell);
+	}
+};
+
+
+var zipEntry = {
+	init : function() {
+		zipEntry.bindEnter();
+	},
+	bindEnter : function() {
+		$(document).keypress(function(e){
+			if (e.which === 13){
```

Comment:
Since no one can ever remember key codes, I'd leave a comment saying what that magic number corresponds to.